layouts/404.html

```diff
@@ -1,3 +1,3 @@
 {{ define "main" }}
-<p style="color: red;">The requested resource could not be found. Please check the address and try again.</p>
+<p style="color: red;">{{ .Param "not-found" }}</p>
```

Comment:
I am not sure, but can't one use `{{ i18b "string_name" }}` for referencing strings in Hugo? If so I think using this would be best, as it is immediately clear that this variable is a translatable variable.

(if it is possible to use the i18n syntax this comment is valid for all usage sites, but I won't comment individually on all occurrences)

Reply:
I did it that way because it was the quickest I found, as I haven't much experience with the multilang part of hugo, but now that the translations where splitted into diferent files, I think it would be feasible to use the i18n syntax after I look into it.